test/scripts/jenkins_ci_group.sh

```diff
@@ -5,7 +5,7 @@ source test/scripts/jenkins_test_setup_oss.sh
 if [[ -z "$CODE_COVERAGE" ]]; then
   checks-reporter-with-killswitch "Functional tests / Group ${CI_GROUP}" yarn run grunt "run:functionalTests_ciGroup${CI_GROUP}";
 
-  if [ "$CI_GROUP" == "1" ]; then
+  if [[ ! "$TASK_QUEUE_PROCESS_ID" && "$CI_GROUP" == "1" ]]; then
```

Comment:
I assume this is and other conditions like it will be removed once we've moved everything over to tasks.

Reply:
Yep, exactly